Fix: issue #116 add comment for test condition6

EdwardNorberg opened the pull request:
Add comments according to test contract. Condition6 correlates to LIC 6

#116

EdwardNorberg committed:
Fix: issue #116 add comment for test condition6

Add comments according to test contract. Condition6 correlates to LIC 6

EdwardNorberg committed:
Fix: issue #117 add comments for test condition7

add comments that explains test contract. Condition7 correlates with LIC
7

EdwardNorberg committed:
Fix: issue #118 add comments for test condition8

add comments for test contracts. Condition8 corresponds to LIC 8

EdwardNorberg committed:
Fix: issue #118 incorrect comment

change the value for B_TPS from 2 to correct value 1 in comments.

EdwardNorberg committed:
Fix: issue #119 add comments for test condition9

Add comments for test contract. Condition9 corresponds to LIC 9.

EdwardNorberg committed:
Fix: issue #121 add comments test condition10

add comments for test contract. Condition10 corresponds with LIC 10.

paulinegarelli reviewed (commented):
There is a typo on the numerotation of the points.
I think you meant either:
"point2 is distance 3 away from line between point1 and point3" if you consider the first point to be point0
or
"point3 is distance 3 away from line between point2 and point4" if you consider the first point to be point1

EdwardNorberg committed:
Fix: issue #122 add comment to test condition11

add comments for test contract. Condition11 corresponds with LIC 11.

paulinegarelli committed:
Merge pull request #160 from kth-cdate-courses/edward/fix/117-comment-test-condition7

Fix: issue #117 add comments for test condition7

paulinegarelli committed:
Merge pull request #162 from kth-cdate-courses/edward/fix/118-comment-test-condition8

Fix: issue #118 add comments for test condition8

EdwardNorberg committed:
Fix: issue #123 add comments for test condition12

add comments for test contract. Condition12 corresponds with LIC 12.

paulinegarelli committed:
Merge pull request #163 from kth-cdate-courses/edward/fix/119-comment-test-condition9

Fix: issue #119 add comments for test condition9

paulinegarelli committed:
Merge pull request #164 from kth-cdate-courses/edward/fix/121-comment-test-condition10

Fix: issue #121 add comments test condition10

paulinegarelli committed:
Merge pull request #165 from kth-cdate-courses/edward/fix/122-comment-test-condition11

Fix: issue #122 add comment to test condition11

paulinegarelli committed:
Merge pull request #166 from kth-cdate-courses/edward/fix/123-comment-test-condition12

Fix: issue #123 add comments for test condition12

EdwardNorberg committed:
Fix: issue #116 fix coments for test condition6

replaced DIST = 5 with DIST = 4 in 2nd test comments, and Point2 wiht Point3 in
both tests comments

EdwardNorberg committed:
Merge branch 'edward/fix/116-comment-test-condtion6' of https://github.com/kth-cdate-courses/DD2480 into edward/fix/116-comment-test-condtion6

EdwardNorberg commented:
Good catch, fixed the comments now.

Storsorken reviewed (approved):
LGTM
